Makefile

```diff
@@ -67,23 +63,16 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
-docker-build: test ## Build docker image with the manager.
-	docker build -t ${IMG} .
```

Comment:
Do we need an equivalent `ko` command to build an image here?

Reply:
`ko apply` will:
- build images
- publish images to the registry
- substitute the image into the yaml
- apply the yaml to the cluster

Reply:
Ok :+1: 

I am new to this tool. Thanks for the explanation.